CHANGES.rdoc

```diff
@@ -3,6 +3,11 @@
 This document describes the relevant changes between releases of the
 _hawkular-client_ project.
 
+=== V 2.7.0
+
+* Support for SSL connections
+* Fix fetching stats to support rates
+
```

Comment:
missing the milestone part like below (Full list of items can be found at https://github.com/hawkular/hawkular-client-ruby/milestone/11?closed=1) but it's not critical (and not a gem requirement)